ci-operator/step-registry/edge-nodes/machineset-manifests/outpost/edge-nodes-machineset-manifests-outpost-ref.yaml

```diff
@@ -0,0 +1,31 @@
+ref:
+  as: edge-nodes-machineset-manifests-outpost
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
+  grace_period: 10m
+  commands: edge-nodes-machineset-manifests-outpost-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: EDGE_ZONE_WORKER_NUMBER
```

Comment:
Per https://github.com/openshift/release/pull/47863#discussion_r1464673872, it should be `EDGE_NODE`, right?

Reply:
Fixed